src/reference-data/defaults/preassigned-fare-products.json

```diff
@@ -407,5 +407,119 @@
         "ATB:UserProfile:1ae560ba"
       ]
     }
+  },
```

Comment:
@oyvindmikkelsenatb Here also ;)

Reply:
@ckbilstad Could you have a look at this as well?

Reply:
@jorelosorio We need to be consistent with the use of «boat» instead of «express boat»